static/app/components/forms/teamSelector.tsx

```diff
@@ -240,7 +240,7 @@ function TeamSelector(props: Props) {
       isOptionDisabled={option => !!option.disabled}
       styles={{
         ...(includeUnassigned ? unassignedSelectStyles : {}),
-        ...placeholderSelectStyles,
+        ...(multiple ? {} : placeholderSelectStyles),
```

Comment:
this is just so we don't show an unnecessary placeholder style for when the multiple prop is true